core/sys/types/src/lib.rs

```diff
@@ -11,3 +11,9 @@
 )]
 
 include!(concat!(env!("OUT_DIR"), "/bindings.rs"));
+
+impl Clone for sgx_key_id_t {
+    fn clone(&self) -> sgx_key_id_t {
+        sgx_key_id_t { id: self.id }
+    }
+}
```

Comment:
This should be done by an `mc_sgx_core_build::SgxParseCallbacks::add_derives()` implementation...

Reply:
@jcape made an attempt in https://github.com/mobilecoinfoundation/sgx/pull/81/commits/bb63bbdd2766b2b5302bf66824b6fa1111f2ad37 let me know thoughts.
I don't like putting *all* the types in the `build/lib.rs`, perhaps `sgx_builder()` should be updated to take a `non-cloneable` iterator.  I don't like negative args, but it seems to be the narrower scope.

Reply:
We should definitely set `derive_copy(false)` in the `sgx_builder()`, that was me simply not thinking.

What we can do is make `SgxParseCallbacks { do_not_clone: Vec<String> }` and `SgxParseCallbacks::new(do_not_clone: impl Iter<Item=&str>)`, and put that into the `Box`.

Reply:
next attempt at, https://github.com/mobilecoinfoundation/sgx/pull/81/commits/a824269f393775f66fd3d7760bdbcd5e37b84a19